README.md

```diff
@@ -589,6 +589,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Jacob Lepler](https://github.com/leplerjacob)
   - [Jade Kneen](https://github.com/jadekneen)
   - [Jagan Sekaran](https://github.com/JAG-010)
+  - [Jagath S M] (https://github.com/JagathSM)
```

Comment:
There's an extra space in here.
```suggestion
  - [Jagath S M](https://github.com/JagathSM)
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient